targets/x11-common

```diff
@@ -70,3 +75,28 @@ ResultAny=no
 ResultInactive=no
 ResultActive=no
 EOF
+
+# Setup default browser
+if [ "${DISTROAKA:-"$DISTRO"}" = 'arch' ]; then
+    install arch=desktop-file-utils,
+
+    mkdir -p /usr/local/share/applications/
+
+    # Application preference: prefer applications that take most effort to be
+    # installed: firefox needs to be installed via pacman, chrome/chromium have
+    # a crouton target, "crouton" needs the extension target, and netsurf is
+    # the fallback installed by gtk-extra.
+    order="firefox.desktop;google-chrome.desktop;chromium.desktop"
+    order="$order;crouton.desktop;netsurf.desktop"
+    mimedefault='/usr/local/share/applications/mimeapps.list'
+    echo "[Added Associations]" > $mimedefault
```

Comment:
Arguably, yes. Users would be better off using `~/.config/mimeapps.list`, though (https://wiki.archlinux.org/index.php/default_applications). I'll replace with an awk script.